src/pagination/pagination-nav/pagination-overflow.component.ts

```diff
@@ -0,0 +1,64 @@
+import {
+	Component,
+	Input,
+	Output,
+	EventEmitter
+} from "@angular/core";
+/**
+ * Used to present a selection of pages when there is an overflow
+ * in the pagination list
+ *
+ * * ```html
+ * <ibm-pagination-overflow [fromIndex]="5" [count]="30" (change)="handleChange(value)"></ibm-pagination-overflow>
+ * ```
+ */
+@Component({
+	selector: "ibm-pagination-overflow",
+	template: `
+		<li class="bx--pagination-nav__list-item" *ngIf="count > 1">
+			<div class="bx--pagination-nav__select">
+			  <select
```

Comment:
There are mixed tabs and spaces in the template indentation which makes it look inconsistent across code editors.
All lines should be indented with tabs only.

Reply:
Sorry my editor tends to misrepresent what I'm looking at vs. whats being saved, if that even sounds possible